neqo-transport/src/connection.rs

```diff
@@ -371,12 +371,12 @@ impl FlowMgr {
         FlowMgr::default()
     }
 
-    pub fn conn_credit_avail(&self) -> u64 {
-        self.max_data - self.used_data
+    pub fn conn_credit_avail(&self) -> usize {
+        (self.max_data - self.used_data) as usize
```

Comment:
How are we managing increases?  I say that because if we have a fixed value for the total amount of credit we are willing to extend, that needs to be a `usize` value.  If that is the case, then we don't need to worry about integer conversion here.  If we are using `u64`, then we might have issues when `self.max_data - self.used_data > usize::max_value()`.